src/Illuminate/Foundation/Console/PresetCommand.php

```diff
@@ -55,7 +55,7 @@ protected function none()
     }
 
     /**
-     * Install the "fresh" preset.
+     * Install the "bootstrap" preset.
```

Comment:
I suppose so. There were two "fresh"-es, I guess it was just copy&pasted and Taylor forgot to change it.